test-projects/mix/test/test_test.exs

```diff
@@ -6,4 +6,8 @@ defmodule TestTest do
     out = Logfmt.encode(foo: "bar")
     assert out == "foo=bar"
   end
+
+  test "ensures ssl" do
+    assert {:ok, _} = Application.ensure_all_started(:ssl)
```

Comment:
@ericmj, would this be enough? I did the same on the `rebar3` test, but it's mostly redundant.

Reply:
Yes, that should be enough.